cloud/blockstore/tools/csi_driver/README.md

```diff
@@ -50,38 +52,60 @@ minikube stop
 minikube start
 ```
 
-# Pull docker image with csi-driver on the node
+# Build csi-driver
+```bash
+./ya make -r cloud/blockstore/tools/csi_driver
+
+```
+
+# Replace symlink with binary
 
 ```bash
-minikube ssh
-docker pull cr.ai.nebius.cloud/crn0l5t3qnnlbpi8de6q/nbs-csi-driver:v0.1
-exit
+DRIVER_PATH=$(readlink -f "./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver")
+rm ./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver
+cp ${DRIVER_PATH} ./cloud/blockstore/tools/csi_driver/cmd/nbs-csi-driver/nbs-csi-driver
+```
+
+# Build docker container with csi-driver
+
+```bash
+eval $(minikube docker-env)
+docker build -t nbs-csi-driver:latest cloud/blockstore/tools/csi_driver
 ```
 
 # Setup k8s cluster
 
 ```bash
+cd cloud/blockstore/tools/csi_driver
 kubectl apply -f ./deploy/manifests/0-ns.yaml
 kubectl apply -f ./deploy/manifests/1-priorityclass.yaml
 kubectl apply -f ./deploy/manifests/2-rbac.yaml
 kubectl apply -f ./deploy/manifests/3-csidriver.yaml
 kubectl apply -f ./deploy/manifests/4-storageclass.yaml
-# Download docker-config.json from the lockbox under the name docker-config.json
-export DOCKER_CONFIG=$(cat docker-config.json)
-kubectl create secret generic nbs-puller-secret \
-    --namespace nbs \
-    --type=kubernetes.io/dockerconfigjson \
-    --from-literal=.dockerconfigjson="$DOCKER_CONFIG"
-kubectl apply -f ./deploy/manifests/5-nbs-configmap.yaml
-kubectl apply -f ./deploy/manifests/6-nbs-daemonset.yaml
-kubectl apply -f ./deploy/manifests/7-csi-deployment.yaml
-kubectl apply -f ./deploy/manifests/8-csi-daemonset.yaml
+kubectl apply -f ./deploy/manifests/5-csi-deployment.yaml
+kubectl apply -f ./deploy/manifests/6-csi-daemonset.yaml
 ```
 
-# copy nbsd and blockstore-nbd to minikube and run them
+# copy nbsd-lightweight, blockstore-nbd and blockstore-client to minikube and run them
 ```bash
-sudo ./nbsd-lightweight --service local --server-file ./server/server.txt --verbose
-sudo ./blockstore-nbd --device-mode endpoint --disk-id my-disk --access-mode rw --mount-mode local --connect-device /dev/nbd0 --listen-path /tmp/nbd.sock
+./ya make -r cloud/blockstore/apps/server_lightweight
+minikube cp cloud/blockstore/apps/server_lightweight/libiconv.so /home/libiconv.so
```

Comment:
You may build with `-DFORCE_STATIC_LINKING=yes` to avoid libraries copying

Reply:
Done. Thanks!